slave/process-k5login/changelog

```diff
@@ -1,3 +1,10 @@
+perun-slave-process-k5login (3.1.6) stable; urgency=low
+
+  * Skip information about not changed k5login files. This information is
+    not important for administrators of a service.
+
+ -- Michal Stava <stavamichal@gmail.com> Wed, 03 Jun 2020 14:57:00 +0100
```

Comment:
Should be +0200, since we currently have CEST

Reply:
Fixed.